extensions/jsonp/runtime/src/main/java/io/quarkus/jsonp/runtime/JsonProviderHolder.java

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.jsonp.runtime;
```

Comment:
I specifically asked for the `io.quarkus.jsonp` package in my comment because I think we will consider this API.

Could you adjust, please?

There is a somewhat unwritten rule that  `io.quarkus.<extension>.runtime` is for internal classes.

Reply:
So this will be considered public API?
In that case it may warrant at least a mention somewhere in the docs. Or at minimum a javadoc comment.